File: KWSiOSSDKObjC/Classes/Services/KWS/Auth/OAuthData.m
//
//  OAuthData.m
//  KWSiOSSDKObjC
//
//  Created by Guilherme Mota on 26/01/2018.
//

#import <Foundation/Foundation.h>
#import "OAuthData.h"


@implementation OAuthData

- (id) initWithCodeVerifier:(NSString *)codeVerifier
             andCodeChallenge:(NSString *)codeChallenge
       andCodeChallengeMethod:(NSString *)codeChallengeMethod{
    
    self = [super init];
    
    _codeVerifier = codeVerifier;
    _codeChallenge = codeChallenge;
    _codeChallengeMethod = codeChallengeMethod;
    
    return self;
    
}


@end

File: KWSiOSSDKObjC/Classes/Base/WebAuth/OAuthData.swift
//
//  OAuthData.swift
//  KWSiOSSDKObjC
//
//  Created by Guilherme Mota on 14/04/2018.
//

import Foundation

public struct OAuthData {
    let codeChallenge: String
    let codeVerifier: String
    let codeChallengeMethod: String
}
